zebedee/client.go

```diff
@@ -192,7 +182,7 @@ func (c *Client) put(ctx context.Context, userAccessToken, path string, payload
 }
 
 // GetBreadcrumb returns a Breadcrumb
-func (c *Client) GetBreadcrumb(ctx context.Context, userAccessToken, collectionID, lang, uri string) ([]Breadcrumb, error) {
```

Comment:
If we do not need these params, should we remove them?

Reply:
yes I think so. However I have reverted most of these 'tidy-ups' as they are introducing breaking changes